src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatabilityAnalyzer.cs

```diff
@@ -97,10 +97,11 @@ public override void Initialize(AnalysisContext context)
             {
                 var typeName = WellKnownTypeNames.SystemOperatingSystem;
 
-                if (!context.Compilation.TryGetOrCreateTypeByMetadataName(typeName + "Helper", out var operatingSystemType))
+                // TODO: remove 'typeName + "Helper"' after tests able to consume the real new APIs
+                if (!context.Compilation.TryGetOrCreateTypeByMetadataName(typeName + "Helper", out var operatingSystemType) &&
+                    !context.Compilation.TryGetOrCreateTypeByMetadataName(typeName, out operatingSystemType))
```

Comment:
Is it possible to write a unit test? I think probably a test without including any OS mock helpers will hit?

Reply:
The type will exist even without Mock helpers or attributes, only if we can add test for the .Net version where this type does not exist, not sure what version that would be, might be netfx